src/components/step-flow/step-flow.stories.tsx

```diff
@@ -167,6 +198,81 @@ export const ExampleImplementation: Story = () => {
 };
 ExampleImplementation.storyName = "Example Implementation";
 
+export const ExampleImplementationWithTitleNode: Story = () => {
+  const lowestStep = 1;
+  const highestStep = 3;
+
+  const [isOpen, setIsOpen] = useState(false);
```

Comment:
comment: sorry @tomdavies73 I've missed this when reviewing and only noticed when I've checked the chromatic build. If we want this to be captured by chromatic we'll need to use [isChromatic](https://github.com/Sage/carbon/blob/ae5079cf2afaff11554a5562e8f0659bb8bcbab0/.storybook/isChromatic.js).